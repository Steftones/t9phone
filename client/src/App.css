html, body {
  width: 100%;
  height: 100%;
  background-color: #f5f9ff;
  margin: 0px;
  padding: 0px;
  overflow-x: hidden; 
}

h1 {
  font-size: 20px;
  margin: 30px 0px 15px 0px;
}

.section {
  padding: 10px;
}

#topSection {
  padding-bottom: 100px;
}

#phoneBody {
  background-color: rgb(17, 37, 82);
  text-align: center;
  height: 370px;
  width: 180px;
  border-radius: 10px;
}

#phoneAntenna {
  background-color: rgb(17, 37, 82);
  height: 80px;
  width: 25px;
  position: relative;
  bottom: 70px;
  left: 160px;
  border-radius: 8px 8px 0px 0px;
}

#phoneTextarea {
  background-color: rgb(168, 167, 167);
  padding: 3px;
  height: 150px;
  width: 135px;
  margin-top: 50px;
  margin-bottom: 5px;
  resize: none;
}

#phoneButtons {
  display: block;
  background-color: rgba(44, 82, 168, 0.589);
  height: 150px;
  width: 135px;
  margin-left: 23px;
  border-radius: 10px;
}

.dialButton {
  width: 40px;
  font-size: 10px;
  border-radius: 20%;
}

