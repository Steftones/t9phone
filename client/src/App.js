import Main from './components/Main';
import NotFound from './components/NotFound';
import { Provider } from 'react-redux';
import store from './redux/store';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import 'bulma/css/bulma.min.css';

const App = () => (
  <Provider store={store}>
    <BrowserRouter>
        <Routes>
          <Route exact path="/*" element={<NotFound />}/>
          <Route exact path="/" element={<Main />}/>
        </Routes>
    </BrowserRouter>
  </Provider>
);

export default App;
