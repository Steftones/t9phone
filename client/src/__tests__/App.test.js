import { render } from "@testing-library/react";
import App from "../App";

test('it should render correctly', () => {
  const { asFragment } = render(<App/>);
  expect(asFragment()).toMatchSnapshot();
});