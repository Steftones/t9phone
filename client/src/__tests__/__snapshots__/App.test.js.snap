// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="column"
      id="topSection"
    />
    <div
      class="columns"
    >
      <div
        class="column is-2"
      />
      <div
        class="column is-flex is-justify-content-center is-4"
      >
        <div
          id="phoneAntenna"
        />
        <div
          id="phoneBody"
        >
          <textarea
            id="phoneTextarea"
            readonly=""
          >
            INPUT: 

OUTPUT: 
          </textarea>
          <div
            id="phoneButtons"
          >
            <button
              class="dialButton"
            >
              1
              <br />
               
            </button>
            <button
              class="dialButton"
            >
              2
              <br />
              abc
            </button>
            <button
              class="dialButton"
            >
              3
              <br />
              def
            </button>
            <button
              class="dialButton"
            >
              4
              <br />
              ghi
            </button>
            <button
              class="dialButton"
            >
              5
              <br />
              jkl
            </button>
            <button
              class="dialButton"
            >
              6
              <br />
              mno
            </button>
            <button
              class="dialButton"
            >
              7
              <br />
              pqrs
            </button>
            <button
              class="dialButton"
            >
              8
              <br />
              tuv
            </button>
            <button
              class="dialButton"
            >
              9
              <br />
              wxyz
            </button>
            <button
              class="dialButton"
            >
              *
              <br />
               
            </button>
            <button
              class="dialButton"
            >
              0
              <br />
               
            </button>
            <button
              class="dialButton"
            >
              #
              <br />
               
            </button>
          </div>
        </div>
      </div>
      <div
        class="column has-text-centered is-4"
      >
        <h1>
          Number to word list converter
        </h1>
        <div
          class="section"
        >
          <button>
            Convert
          </button>
            
          <button>
            Reset
          </button>
        </div>
        <div
          class="section"
        >
          Mode:
          <br />
          <div
            class="select"
          >
            <select>
              <option>
                Decrypt T9
              </option>
              <option>
                Filter words
              </option>
            </select>
          </div>
        </div>
      </div>
      <div
        class="column is-2"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`should render the component 1`] = `
<DocumentFragment>
  <div>
    <div
      class="column"
      id="topSection"
    />
    <div
      class="columns"
    >
      <div
        class="column is-2"
      />
      <div
        class="column is-flex is-justify-content-center is-4"
      >
        <div
          id="phoneAntenna"
        />
        <div
          id="phoneBody"
        >
          <textarea
            id="phoneTextarea"
            readonly=""
          >
            INPUT: 

OUTPUT: 
          </textarea>
          <div
            id="phoneButtons"
          >
            <button
              class="dialButton"
            >
              1
              <br />
               
            </button>
            <button
              class="dialButton"
            >
              2
              <br />
              abc
            </button>
            <button
              class="dialButton"
            >
              3
              <br />
              def
            </button>
            <button
              class="dialButton"
            >
              4
              <br />
              ghi
            </button>
            <button
              class="dialButton"
            >
              5
              <br />
              jkl
            </button>
            <button
              class="dialButton"
            >
              6
              <br />
              mno
            </button>
            <button
              class="dialButton"
            >
              7
              <br />
              pqrs
            </button>
            <button
              class="dialButton"
            >
              8
              <br />
              tuv
            </button>
            <button
              class="dialButton"
            >
              9
              <br />
              wxyz
            </button>
            <button
              class="dialButton"
            >
              *
              <br />
               
            </button>
            <button
              class="dialButton"
            >
              0
              <br />
               
            </button>
            <button
              class="dialButton"
            >
              #
              <br />
               
            </button>
          </div>
        </div>
      </div>
      <div
        class="column has-text-centered is-4"
      >
        <h1>
          Number to word list converter
        </h1>
        <div
          class="section"
        >
          <button>
            Convert
          </button>
            
          <button>
            Reset
          </button>
        </div>
        <div
          class="section"
        >
          Mode:
          <br />
          <div
            class="select"
          >
            <select>
              <option>
                Decrypt T9
              </option>
              <option>
                Filter words
              </option>
            </select>
          </div>
        </div>
      </div>
      <div
        class="column is-2"
      />
    </div>
  </div>
</DocumentFragment>
`;
